appeals/api/src/server/mappers/integration/commands/service-user.mapper.js

```diff
@@ -14,7 +16,8 @@ export const mapServiceUserIn = (data) => {
 			webAddress: data.website,
 			phoneNumber: data.telephoneNumber,
 			otherPhoneNumber: data.otherPhoneNumber,
-			faxNumber: data.faxNumber
+			faxNumber: data.faxNumber,
+			address: { create: mapAddressIn(data) }
```

Comment:
see comment above - will this break other uses of service user mapper outside representations?

Reply:
it was breaking some other tests where it was being used so have split it into own function